api/swagger.yaml

```diff
@@ -6858,6 +6858,42 @@ definitions:
               example: "24"
       FirewallBackend:
         $ref: "#/definitions/FirewallInfo"
+      DiscoveredDevices:
```

Comment:
I'm usually for agnostic names, but in this case I see that we named the field for spec-dirs `CDISpecDirs`, so I'm wondering if we should name this fields somewhat matching that (`CDI` prefix?)

Reply:
Or do we expect this to be re-purposed for other types of devices?

Reply:
The intent was to make this non-CDI specific (following original suggestion to also support it for `nvidia` driver).

Reply:
Gotcha, thanks! Yes, would need to have my brain go about what kind of devices we would have here.